src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -0,0 +1,27 @@
+from argparse import ArgumentParser
+
+import pytest
+
+from diffpy.labpdfproc.tools import load_user_metadata
+
+params5 = [
+    ([[]], []),
+    ([["toast=for breakfast"]], [["toast", "for breakfast"]]),
+    ([["mylist=[1,2,3.0]"]], [["mylist", "[1,2,3.0]"]]),
```

Comment:
I think this is ok since we are just writing the metadata to a file, but actually what i had in mind for the output was `["mylist", [1,2,3.0]]` not `["mylist", "[1,2,3.0]"]`.